smslibrary/src/test/java/com/eis/smslibrary/SMSMessageHandlerTest.java

```diff
@@ -0,0 +1,46 @@
+package com.eis.smslibrary;
+
+import com.eis.communication.MessageParseStrategy;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author Luca Crema
+ */
+public class SMSMessageHandlerTest {
+
+    SMSMessageHandler defaultInstance;
```

Comment:
_private static_, no?

Reply:
I can agree on private (even though it's just a test so nobody cares about encapsulation) but **why static**?

Reply:
Ok it doesn't have to be a **static** object. Only the instance within the class _SMSMessageHandler_.
My mistake, excuse my teacher....